contentful/apps
# feat: add empty state for channel selection and teams app help text [INTEG-1555]

## Purpose

This PR adds an empty state for the channel selection modal and help text under the Tenant Id field to prompt users to install the Contentful MS Teams app.

## Approach

Added our `integration-component-library` as a dependency to utilize the `HyperLink` component. For now the link to Teams just brings you to Teams, but eventually we can update that to go directly to our app.

Empty State
![Screenshot 2023-11-28 at 4 02 15 PM](https://github.com/contentful/apps/assets/62958907/63b3a721-35dc-4a8f-86ed-52e0a15457ff)

Help Text
![Screenshot 2023-11-29 at 8 53 17 AM](https://github.com/contentful/apps/assets/62958907/32ca02f1-9c2e-4cf1-9a7e-eebdb91634f0)

## Testing steps

Open the MS Teams development app. Change the mock channels so that it is an empty array in order to see the empty state.

## Breaking Changes

## Dependencies and/or References

## Deployment
